datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+// [START datacatalog_create_fileset_tag]
+
+import com.google.api.gax.rpc.AlreadyExistsException;
+import com.google.cloud.datacatalog.ColumnSchema;
+import com.google.cloud.datacatalog.CreateEntryGroupRequest;
+import com.google.cloud.datacatalog.CreateEntryRequest;
+import com.google.cloud.datacatalog.Entry;
+import com.google.cloud.datacatalog.EntryGroup;
+import com.google.cloud.datacatalog.EntryType;
+import com.google.cloud.datacatalog.GcsFilesetSpec;
+import com.google.cloud.datacatalog.LocationName;
+import com.google.cloud.datacatalog.Schema;
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+
+import java.io.IOException;
+
+public class CreateFilesetEntry {
+
+  /**
+   * Create Fileset Entry.
+   *
+   * @param projectId  e.g. 'my-project'
+   * @param entryGroupId  e.g. 'fileset_entry_group'
+   * @param entryId e.g. 'fileset_entry_id'
+   */
+  public static void createEntry(String projectId, String entryGroupId, String entryId) {
+    // Currently, Data Catalog stores metadata in the
+    // us-central1 region.
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DataCatalogClient dataCatalogClient = DataCatalogClient.create()) {
+      // 1. Create an Entry Group.
```

Comment:
So normally with snippets like this, we are only trying to showcase how to do a single action. Is there a particular reason you have grouped `createEntryGroup` and `createEntry ` together? Normally I would expect a different snippet for each (since I'm assuming you can have multiple entries in a group).

Reply:
Your assumption is right, you can have multiple entries in a group. The first time a user creates a fileset entry, he must have created the Entry Group before... But they could be in separe files. Do you want me to split them in two files in this PR or open a separate PR?

Reply:
I'm okay with 2 files.

The user is most likely to see this in the context of our docs, so you will likely have the snippets references in different locations.

Reply:
Changed to 2 files.